src/python/bcc/__init__.py

```diff
@@ -1093,7 +1093,12 @@ def cleanup(self):
             # non-string keys here include the perf_events reader
             if isinstance(k, str):
```

Comment:
I would love to see we separate storage for kprobes and perf ring buffers. But probably would do it in a separated change and after #1197 is discussed:)